docs/services/data/_index.md

```diff
@@ -268,9 +269,10 @@ The following attributes are available for the data management service:
 | `sync_interval_mins` | float | Optional | Time interval in minutes between syncing to the cloud. Viam does not impose a minimum or maximum on the frequency of data syncing. However, in practice, your hardware or network speed may impose limits on the frequency of data syncing. <br> Default: `0.1`, meaning once every 6 seconds. |  <p class="center-text"><i class="fas fa-check" title="yes"></i></p> |
+| `delete_data_on_part_deletion` | bool | Optional | Whether deleting this machine part or this machine should result in deleting all the data captured by that machine part. <br> Default: `false` | <p class="center-text"><i class="fas fa-check" title="yes"></i></p> |
 | `delete_every_nth_when_disk_full` | int | Optional | How many files to delete when local storage meets the [fullness criteria](/services/data/#storage). The data management service will delete every Nth file that has been captured upon reaching this threshold. Use JSON mode to configure this attribute. <br> Default: `5`, meaning that every fifth captured file will be deleted. |  <p class="center-text"><i class="fas fa-times" title="no"></i></p> |
 | `maximum_num_sync_threads` | int | Optional | Max number of CPU threads to use for syncing data to the Viam Cloud. <br> Default: [runtime.NumCPU](https://pkg.go.dev/runtime#NumCPU)/2 so half the number of logical CPUs available to viam-server |  <p class="center-text"><i class="fas fa-times" title="no"></i></p> |
-| `mongo_capture_config.uri` | string | Optional | The [MongoDB URI](https://www.mongodb.com/docs/v6.2/reference/connection-string/) data capture will attempt to write tabular data to after it is enqueued to be written to disk. When non empty, data capture will capture tabular data to the configured MongoDB database & collection at that URI.<br>See  `mongo_capture_config.database` and  `mongo_capture_config.collection` below for database & collection defaults.<br>See [Data capture Directly To MongoDB](/services/data/#capture-directly-to-mongodb) for an example config.|  <p class="center-text"><i class="fas fa-times" title="no"></i></p> |
+| `mongo_capture_config.uri` | string | Optional | The [MongoDB URI](https://www.mongodb.com/docs/v6.2/reference/connection-string/) data capture will attempt to write tabular data to after it is enqueued to be written to disk. When non empty, data capture will capture tabular data to the configured MongoDB database & collection at that URI.<br>See  `mongo_capture_config.database` and  `mongo_capture_config.collection` below for database & collection defaults.<br>See [Data capture directly To MongoDB](/services/data/#capture-directly-to-mongodb) for an example config.|  <p class="center-text"><i class="fas fa-times" title="no"></i></p> |
```

Comment:
```suggestion
| `mongo_capture_config.uri` | string | Optional | The [MongoDB URI](https://www.mongodb.com/docs/v6.2/reference/connection-string/) data capture will attempt to write tabular data to after it is enqueued to be written to disk. When non-empty, data capture will capture tabular data to the configured MongoDB database and collection at that URI.<br>See  `mongo_capture_config.database` and  `mongo_capture_config.collection` below for database and collection defaults.<br>See [Data capture directly to MongoDB](/services/data/#capture-directly-to-mongodb) for an example config.|  <p class="center-text"><i class="fas fa-times" title="no"></i></p> |
```
Noticed more nits before realizing they were out-of-diff; might as well change. Other option is "not empty"

Reply:
Thanks, this is helpful!